overlay/frameworks/base/core/res/res/values/config.xml

```diff
@@ -264,7 +264,7 @@
     <bool name="config_cellBroadcastAppLinks">true</bool>
 
     <!-- Is the device capable of hot swapping an ICC Card -->
-    <bool name="config_hotswapCapable">true</bool>
+    <bool name="config_hotswapCapable">false</bool>
```

Comment:
I'm pretty sure this change is not needed. We explicitely handle the `ACTION_CARRIER_CONFIG_CHANGED` event which triggers `evaluateCarrierBundle` that prompts for a reboot if required. With this change Android will always prompt for a reboot

Reply:
> After a SIM is removed, the modem dies completely, so there will be no more SIM to evaluate a carrier for, the device must be rebooted.

I just tried and you are right. Didn't expect that but that's how it is. So this change is ok then.

> Even if the modem didn't die completely, we now have only one Carrier Config applied for a bunch of operators.

I don't understand that. If we need a different Carrier Config it will/would be applied and prompt for a reboot. Doesn't matter though as due to the modem bug above we can't hotswap...